platform/o.n.html.ko4j/external/ko4j-1.8.1-license.txt

```diff
@@ -1,7 +1,7 @@
 Name: Knockout4J
 Version: 1.8.1
 Description: Knockout4J
-License: Apache-2.0
+License: Apache-2.0-ko4j
```

Comment:
Updating the ko4j license to the version that is inside the binary - includes license terms for the Knockout library, in addition to the Apache 2.0 terms.